test/spec/controllers/menuCtrlSpec.js

```diff
@@ -3,24 +3,83 @@
 describe('Menu Controller', function() {
 
   describe('MenuCtrl', function(){
-    var scope, ctrl, user, $httpBackend, $window;
+    var scope, ctrl, user, User, $httpBackend, $window, rootScope;
 
     beforeEach(function(){
       module(function($provide) {
         $provide.value('Chat', { seenMessage: function() {} });
+        user = specHelper.newUser();
+        user._id = "unique-user-id";
+        User = {
+          sync: sandbox.stub(),
+          set: sandbox.stub(),
+          user: user,
+        }
+        $provide.value('User', User);
       });
 
-      inject(function(_$httpBackend_, $rootScope, $controller) {
+      inject(function(_$httpBackend_, $rootScope, $controller, _$rootScope_) {
```

Comment:
We should just have one $rootScope instance. It's customary to use the underline syntax for tests, so go with that one. You could do the same for `_$controller_`